electrumsv/wallet_database/functions.py

```diff
@@ -2075,6 +2077,61 @@ def _write(db: Optional[sqlite3.Connection]=None) -> None:
     return db_context.post_to_thread(_write)
 
 
+async def read_reorged_transactions_async(db_context: DatabaseContext,
+        orphaned_block_hashes: List[bytes]) -> List[bytes]:
+    return await db_context.run_in_thread_async(_read_reorged_transactions,
+        orphaned_block_hashes=orphaned_block_hashes)
+
+
+def _read_reorged_transactions(db: Optional[sqlite3.Connection]=None,  # pylint: disable=dangerous-default-value
```

Comment:
I do not believe this works. The `db` argument for write thread callbacks now comes last due to Python typing limitations and should not require a pylint disable. This should be a direct read as per last comment.